### ISSUE TITLE
Startup Issue

### ISSUE THREAD

**Issue by TSpeedman** [bisq-network/support#1659]
I have been trying for about four days to open bisq, but am getting stuck on the below loading screen. I am using a Mac (Sonoma 14.6) and have updated Bisq to the latest version. I run my own umbrel node, which I believe bisq is pointed to - have never had issues in the past.

![image](https://github.com/user-attachments/assets/0b81a9a1-6114-45e0-ad06-82e761b9d0f2)

![image](https://github.com/user-attachments/assets/32b1172e-4552-4720-a1da-e097434490b3)

I seem to be stuck at 2 Bitcoin network peers and am not sure how I can fix this. I have deleted the SPV chain file and get the same message (above) each time I open the application. I've also deleted and re-installed the application a few times, restarted computer, still no luck.

Lastly, I have left bisq open for more than 24 hours and it just stays on the above screens.

Any help would be appreciated.

**Comment by darawhelan** [comment]
@TSpeedman 

Are you running the latest version of Bisq? You should be using 1.9.17. Older versions will no longer connect.

Are you connecting over Tor or do you have the application set up to connect over clear net?

**Comment by TSpeedman** [comment]
Yes, I am running the latest version. Below are my TOR settings:

![image](https://github.com/user-attachments/assets/4078eec3-dec0-48e1-bceb-7967ac2e2d34)

**Comment by TSpeedman** [comment]
I can't see my bisq network settings because the application won't open.

**Comment by darawhelan** [comment]
I would follow the instructions here: https://bisq.wiki/Last_resort_restore

If you set up a new instance of Bisq it should connect. You can then import your wallet / payment profiles etc into the new instance.

**Comment by TSpeedman** [comment]
@darawhelan this is very helpful - thank you.

I followed the steps and was able to get Bisq to start up properly. My payment accounts, etc. were all there as expected but I was unable to connect to the bitcoin network. I noticed that by default I was connected to a public node (whatever the default setting is). I changed it to my node, and hit the resync SPV button, then closed the application.

I've now restarted and it looks like i've got 4-6 bitcoin network peers, as opposed to the 2 I was stuck with previously. Nice!

From what I understand this can take a while (a few hours maybe?) so I am going to leave it and see what happens. If nothing works, I will just start from scratch, then stay connected to public nodes, resync the SPV and hopefully everything is good.

Let me know if I am doing anything wrong or you have any other suggestions.

Thank you! Will report back in a few hours or tomorrow morning.
